lib/index.js

```diff
@@ -125,27 +123,56 @@ function junkFoodMove(junkFood){
  junkFood.y += junkFood.speed;
 }
 
-function isHealthFoodCollidingWithTray(healthFood, tray){
-    return !(
-      tray.x > healthFood.x + healthFood.width ||
-      tray.x + tray.width < healthFood.x ||
-      tray.y > healthFood.y + healthFood.height ||
-      tray.y + tray.height < healthFood.y);
-}
+window.setInterval(collisionOutcome, 200, healthFoods, world.tray);
 
 function collisionOutcome(healthFoods, tray){
-  for (var i = 0; i < healthFoods.length; i++)
-  {var healthFood = healthFoods[i];
+  for (var i = 0; i < healthFoods.length; i++){
+    var healthFood = healthFoods[i];
```

Comment:
I wonder if this is something we could do with junkFood in the same function too?